### PR TITLE
Group constants

### PR DESCRIPTION
**Pull request by Gandi24**
Introducing first attempt to parametrize meetups.

### CODE REVIEW

File: pygroups/templates/meetups/meetup_promo.html [hunk 1]
```diff
@@ -96,16 +96,16 @@ <h2>YouTube</h2>
...
                         {{s.first_name}} {{s.last_name}}{% if not forloop.last %}, {% endif %}
                         {% endfor %}<br/>
                     <br/>
                     {{ talk.description }}<br/>
                     <br/>
-                    Prezentacja odbyła się podczas spotkania Pykonik #{{meetup.number}} ({{meetup.date|date:"d.m.Y"}}) - http://www.pykonik.org/{{meetup.number}}/<br/>
+                    Prezentacja odbyła się podczas spotkania {{ group.name }} #{{meetup.number}} ({{meetup.date|date:"d.m.Y"}}) - {{ group.page_address }}{{meetup.number}}/<br/>
                     <br/>
-                    Pykonik to spotkania krakowskiej społeczności Pythonistów. Głównym założeniem inicjatywy jest wymiana doświadczeń w trybie face2face i przelanie energii pasjonatów tego dynamicznego języka programowania w konkretne działania dla dobra ludzkości.
+                    {{ group.name }} to spotkania krakowskiej społeczności Pythonistów. Głównym założeniem inicjatywy jest wymiana doświadczeń w trybie face2face i przelanie energii pasjonatów tego dynamicznego języka programowania w konkretne działania dla dobra ludzkości.
```
Comment (artcz) [inline pygroups/templates/meetups/meetup_promo.html]: `{{ group.name }} (...) krakowskiej`

File: pygroups/templates/misc/faq.html [hunk 2]
```diff
@@ -53,68 +53,68 @@ <h3>Jaki rodzaj sponsoringu preferujemy?</h3>
...
             <div id="description-pl" class="modal hide fade" tabindex="-1" role="dialog" aria-hidden="true">
                 <div class="modal-header">
                     <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    <h3>Opis Pykonik - PL</h3>
+                    <h3>Opis {{ group.name }} - PL</h3>
                 </div>
                 <div class="modal-body">
                     <p>
-                        Pykonik to spotkania krakowskiej społeczności Pythonistów. Głównym założeniem inicjatywy jest wymiana doświadczeń w trybie face2face i przelanie energii pasjonatów tego dynamicznego języka programowania w konkretne działania dla dobra ludzkości.
+                        {{ group.name }} to spotkania krakowskiej społeczności Pythonistów. Głównym założeniem inicjatywy jest wymiana doświadczeń w trybie face2face i przelanie energii pasjonatów tego dynamicznego języka programowania w konkretne działania dla dobra ludzkości.
```
Comment (artcz) [inline pygroups/templates/misc/faq.html]: ditto

File: pygroups/templates/misc/faq.html [hunk 3]
```diff
@@ -53,68 +53,68 @@ <h3>Jaki rodzaj sponsoringu preferujemy?</h3>
...
 
             <div id="description-en" class="modal hide fade" tabindex="-1" role="dialog" aria-hidden="true">
                 <div class="modal-header">
                     <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    <h3>Opis Pykonik - EN</h3>
+                    <h3>Opis {{ group.name }} - EN</h3>
                 </div>
                 <div class="modal-body">
-                    <p>Pykonik is a community that gathers Python hobbyists from Krakow and its surroundings.</p>
+                    <p>{{ group.name }} is a community that gathers Python hobbyists from Krakow and its surroundings.</p>
```
Comment (artcz) [inline pygroups/templates/misc/faq.html]: Kraków

File: pygroups/templates/misc/home.html [hunk 4]
```diff
@@ -125,15 +125,15 @@ <h3>spotkanie w przygotowaniu</h3>
...
+                <h2>{{ group.city_adjective }} grupa użytkowników Pythona</h2>
                 <div class="row">
                   <div class="video-landing">
 {#                    <iframe width="800" height="410" src="http://www.youtube.com/embed/Rrbpnrs8neM" frameborder="0" allowfullscreen></iframe>#}
                   </div>
                 </div>
                 <div class="row">
                     <div class="span5">
-                        <p> Pykonik to spotkania krakowskiej społeczności Pythonistów.
+                        <p> {{ group.name }} to spotkania krakowskiej społeczności Pythonistów.
```
Comment (artcz) [inline pygroups/templates/misc/home.html]: krakowskiej

File: pygroups/templates/base.html [hunk 5]
```diff
@@ -3,16 +3,16 @@
...
-    <meta name="description" content="Pykonik to cykliczne spotkania pasjonatów języka programowania Python z Krakowa i okolic.">
+    <title>{{ group.name }} | {% block title %}{% endblock %}</title>
+    <meta name="keywords" content="{{ group.name }}, Python, grupa użytkowników, programowanie, barcamp, spotkanie, prezentacje">
+    <meta name="description" content="{{ group.name }} to cykliczne spotkania pasjonatów języka programowania Python z {{ group.city_genitive }} i okolic.">
 
     <meta property="og:type" content="website">
-    <meta property="og:title" content="Pykonik | {% block og_title %}{% endblock %}"/>
-    <meta property="og:description" content="Pykonik to cykliczne spotkania pasjonatów języka programowania Python z Krakowa i okolic.">
+    <meta property="og:title" content="{{ group.name }} | {% block og_title %}{% endblock %}"/>
+    <meta property="og:description" content="{{ group.name }} to cykliczne spotkania pasjonatów języka programowania Python z Krakowa i okolic.">
```
Comment (artcz) [inline pygroups/templates/base.html]: Krakowa

**Comment by Gandi24** [comment]
@artcz Fix done

File: pygroups/pygroups/settings/pygroups_secret_variables/base.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (artcz) [inline pygroups/pygroups/settings/pygroups_secret_variables/base.json]: should this be in the repository? (it's a file added by the provision-env script)

File: pygroups/templates/meetups/emails/talk_proposal_subject.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+[{{ page_address }}] Zgłoszenie prezentacji - {{ talk_proposal.talk.title }} ({{ talk_proposal.talk.speakers.all.0 }})
```
Comment (artcz) [inline pygroups/templates/meetups/emails/talk_proposal_subject.txt]: why .0?

**Pull request merged**